ajax/libs/sticky-kit/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sticky-kit",
+  "version": "1.1.3",
+  "description": "A jQuery plugin for creating smart sticky elements",
+  "filename": "sticky-kit.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/leafo/sticky-kit.git"
+  },
+  "author": {
+    "name": "leafo",
+    "email": "leafot@gmail.com"
+  },
+  "license": "MIT",
+  "keywords": [
+    "sticky",
+    "fixed",
+    "scrolling",
+    "ui"
+  ],
+  "homepage": "https://leafo.net/sticky-kit/",
+  "npmName": "sticky-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.+(js|map|css)"
```

Comment:
`**/*.+(js|map|css)` might be a better choice? Because it can get every file in sub-folders.